sahara.c

```diff
@@ -196,8 +196,8 @@ static int sahara_done(struct qdl_device *qdl, struct sahara_pkt *pkt)
 
 int sahara_run(struct qdl_device *qdl, char *prog_mbn)
 {
-	struct sahara_pkt *pkt;
-	char buf[4096];
+	struct sahara_pkt *pkt = NULL;
```

Comment:
these 2 lines are not related to commit log.

Reply:
I have updated patch with removed extra initialization.